scan/lib/scan/options.rb

```diff
@@ -223,7 +223,9 @@ def self.available_options
                                      description: "Create an Incoming WebHook for your Slack group to post results there",
                                      optional: true,
                                      verify_block: proc do |value|
-                                       UI.user_error!("Invalid URL, must start with https://") unless value.start_with? "https://"
+                                       if !value.to_s.empty? && !value.start_with?("https://")
```

Comment:
In further conversation, we decided to leave the action's implementation as it is 👍